client/app/components/packages/attachments.js

```diff
@@ -22,6 +20,11 @@ export default class PackagesAttachmentsComponent extends Component {
     this.fileManager.unMarkFileForDeletion(file);
   }
 
+  @action
```

Comment:
I'd just add a comment something along the lines of this so future developers can understand why this action isn't being passed any files as parameters (because it doesn't need to know, just incremenets the variable by one to trigger `isDirty`).
```js
 // we need this to track is the attachments isDirty, action just increments a variable by one so we can respond in UI
  @action
  trackFileForUpload() {
    this.fileManager.trackFileForUpload();
  }
```